src/Glpi/Application/View/TemplateRenderer.php

```diff
@@ -91,18 +92,18 @@ public function __construct(string $rootdir = GLPI_ROOT, string $cachedir = GLPI
             || (!file_exists($tpl_cachedir) && !is_writable($cachedir))
         ) {
             trigger_error(sprintf('Cache directory "%s" is not writeable.', $tpl_cachedir), E_USER_WARNING);
-        } else {
+        } else if (($_SESSION['glpi_use_mode'] ?? null) !== Session::DEBUG_MODE) {
```

Comment:
If you want to disable cache on dev env, then the correct solution is to check the GLPI_ENVIRONMENT_TYPE constant.
```suggestion
        } else if (GLPI_ENVIRONMENT_TYPE !== GLPI::ENV_DEVELOPMENT) {
```
However, the `auto_reload` should ensure that cache is not used when the source file is changed.

Did you have the `GLPI_ENVIRONMENT_TYPE` constant correctly defined to `development` in your local env ?

Reply:
I think I had, but I cannot find docs about setting it, so I'm not sure.